types.d.ts

```diff
@@ -0,0 +1,8 @@
+import { FeatureCollection, Feature, GeometryObject } from "geojson";
+
+export type Intersection = [number, number];
+
+export default function sweeplineIntersections(
+  geojson: FeatureCollection<GeometryObject> | Feature<GeometryObject>,
```

Comment:
@mfedderly perhaps you had reasons, but should should geojson param also accept GeometryObject, not just Feature and FeatureCollection.  Also it's not all GeometryObject types according to the docs?, just `LineString | MultiLineString | Polygon | MultiPolygon`.  So the signature would be something like:

```
geojson: FeatureCollection<LineString | MultiLineString | Polygon | MultiPolygon> | Feature<LineString | MultiLineString | Polygon | MultiPolygon> | LineString | MultiLineString | Polygon | MultiPolygon
```

See docs:
"Valid inputs: Geojson Feature or Geometry including Polygon, LineString, MultiPolygon, MultiLineString, as well as FeatureCollection."

Reply:
The second argument `ignoreSelfIntersection` also should be optional according to the docs

Reply:
You are right @twelch 😂  - my review clearly stank!

Reply:
good catch @twelch !